t/ppi_element_replace.t

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/perl
+
+# Unit testing for PPI::Element
+
+use lib 't/lib';
+use PPI::Test::pragmas;
+
+use PPI::Document ();
+use Test::More;
+
+__REPLACE: {
+	my $Document = PPI::Document->new( \"print 'Hello World';" );
+	isa_ok( $Document, 'PPI::Document' );
+	my $string = $Document->find_first('Token::Quote');
+	isa_ok( $string, 'PPI::Token::Quote' );
+	is( $string->content, "'Hello World'", 'Got expected token' );
+	my $foo = PPI::Token::Quote::Single->new("'foo'");
+	isa_ok( $foo, 'PPI::Token::Quote::Single' );
+	is( $foo->content, "'foo'", 'Created Quote token' );
+	$string->replace( $foo );
+	is( $Document->serialize, "print 'foo';", 'replace works' );
+}
+
+__REPLACE_PARENT: {
```

Comment:
this had me confused for a little bit, does it make sense to rename these after what they're really testing, i.e.

__REPLACE_METH and __REPLACE_CHILD_METH ?